build.rs

```diff
@@ -1,136 +1,14 @@
-use std::convert::TryFrom;
-use std::{env, path::PathBuf};
+#[cfg(not(any(feature = "pio", feature = "native")))]
+compile_error!("One of the features `platformio` or `native` must be selected.");
```

Comment:
Rename to `pio` to match the `cargo-pio` wording etc.
Also currently the code does not compile (with pio) as half of the places refer to the feature as `pio` (see `#[cfg(not(any(feature = "pio", feature = "native")))]`) and the other half - as `platformio`